data/src/languages_update.py

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env python
+"""languages.json update
 
+Takes languages.json file and updates the script entry for a language based on
+the files that are present in data/tsv.
+"""
 import collections
 import json
 import operator
 import os
-import sys
-
 import unicodedataplus
+
+from typing import Tuple, Union, DefaultDict, Dict
```

Comment:
Can you sort the imports? i.e.

   from typing import DefaultDict, Dict, Tuple, Union